src/main/resources/assets/parcool/book/parcool_guide_content_en_us.txt

```diff
@@ -17,24 +17,28 @@ There is another similar word, {I}'Free Running'{reset}. This is said to be a wo
 This mod add a concept of Stamina.
 Stamina is consumed when you use ParCool Actions.
 If you consumed all stamina,
-1.You can’t use ParCool Actions.
-2.You can’t sprint.
+1.[You can’t use ParCool Actions]{B}
```

Comment:
If you want to show this like following text,
1.**You can't use ParCool Actions**
You should rewrite it like,
1.{B}You can't use ParCool Actions{reset}